include/pistache/date.h

```diff
@@ -4370,7 +4371,7 @@ scan_keyword(std::basic_istream<CharT, Traits>& is, FwdIter kb, FwdIter ke)
     const unsigned char does_match = '\2';
     unsigned char statbuf[100];
     unsigned char* status = statbuf;
-    unique_ptr<unsigned char, void(*)(void*)> stat_hold(0, free);
+    std::unique_ptr<unsigned char, void(*)(void*)> stat_hold(0, free);
```

Comment:
Note: It wasn't compiling without `std` for me

Reply:
I have not studied this file much, but no header should ever import a namespace, not even std.  All symbols in headers should be fully qualified. Adding 'std::' here was the correct thing to do.